### PR TITLE
onRowsChanged isn't guaranteed to fire when filtered items change.

### PR DESCRIPTION
**Pull request by albertywu**
Add an explicit event onFilteredItemsChanged event to capture this.

### CODE REVIEW

File: slick.dataview.js [hunk 1]
```diff
@@ -873,6 +875,11 @@
         onRowCountChanged.notify({previous: countBefore, current: rows.length}, null, self);
         onRowsChanged.notify({rows: diff}, null, self);
       }
+      if (filteredItems.length != previousFilteredItems.length) {
```
Comment (albertywu) [inline slick.dataview.js]: keeping the "change" definition consistent with `onRowsChanged` (looking at length). Might want to do a tech debt later to fix this in both places (so we can pick up changes if array indices swap for example).

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "SlickGrid",
-  "version": "3.9.1",
+  "version": "3.9.2",
```
Comment (ghost) [inline package.json]: could version bumps be done in a separate commit?  makes it easier to see in the history

Comment (albertywu) [inline package.json]: sure

File: slick.dataview.js (hunk 1 above)
Comment (ghost) [inline slick.dataview.js]: FWIW, `!==`

**Comment by ghost** [comment]
LGTM, but it says it has conflicts - maybe you need to rebase?
